runtime/autoload/dist/ft.vim

```diff
@@ -263,6 +263,19 @@ export def EuphoriaCheck()
   endif
 enddef
 
+export def SetExeclineFiletype()
+  if getline(1) =~# '^#!.*\<execlineb\>'
+    # shebang includes execlineb
+    setf execline
+  elseif expand('%:p') =~# 's6' && expand('%:t') =~# '^\(up\|down\|run\|finish\)$'
+    # path includes s6 somewhere, and filename is exactly one of the expected service runners
+    setf execline
+  elseif expand('%:t') =~# '^s6-'
+    # filename starts with s6-
+    setf execline
+  endif
```

Comment:
is it preferred to keep it in `filetype.vim`, instead of moving it into a standalone script? i moved it out mainly to keep `filetype.vim` as tidy as possible.

Reply:
oh, that's actually quite clean. thanks, i'll change it :)

Reply:
Quick question: Does `*s6*/up` include a path such as `/etc/s6/sub/path/up` too? I imagine the `*` includes `/` too, but better to be safe than sorry.